types/asmresult/asmresult.interfaces.ts

```diff
@@ -37,7 +37,7 @@ export type ParsedAsmResult = {
     parsingTime?: string;
     filteredCount?: number;
     externalParserUsed?: boolean;
-    objdumpTime?: number;
+    objdumpTime?: number | string;
```

Comment:
Apparently it's set to a string in one of the compilers

Reply:
Oh that's odd - which compiler? Seems like it ought not to be (maybe a TODO?)

Reply:
`java` was the only compiler erroring however other compilers do similar assignments, e.g. clang does `result.objdumpTime = objResult.execTime;` but `result` is `any` (`objResult.execTime` is a string). I'm not sure how this type is used throughout the codebase, I'll go ahead and add a TODO.

Reply:
Its set to a string, but the frontend always takes that into account. There was a good reason for it, but I really can’t remember why. But it is what it is. We can change it later on when we have a clear view of what everything is and should be